src/constraint_solver/io.cc

```diff
@@ -180,6 +188,16 @@ class FirstPassVisitor : public ModelVisitor {
   }
 
  private:
+  //// TODO: TBD: not sure we are confusing terms with IntExpr, losing IntVar-iness, nevermind Boolean, etc...
```

Comment:
Can you copy paste which test shows that?

Not delegating is years of work, and lots of slowdown in the solver :-) So it won't change :-)